tests/Integration/IgnoreGettersTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Yajra\DataTables\Tests\Integration;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Yajra\DataTables\DataTables;
+use Yajra\DataTables\Tests\Models\User;
+use Yajra\DataTables\Tests\TestCase;
+
+class IgnoreGettersTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /** @test */
+    public function it_return_the_default_value_when_attribute_is_null()
+    {
+        $user = User::create([
+                'name'  => 'foo',
+                'email' => 'foo@bar.com',
+                'color' => null
+            ]);
+
+        $this->assertEquals('#000000', $user->color);
+        $this->assertEquals('#000000', $user->refresh()->toArray()['color']);
+    }
+
+    /** @test */
+    public function it_return_the_getter_value_without_ignore_getters()
```

Comment:
Sorry for not clearing my request. The assertion is on this test that when ignore_getters are not set, the computed attribute would still be displayed.